src/gxcore/sd/interop/canOpen.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Checks if an URL can be opened
+ * Current implementation only returns `true` for http and https URLs
+ * @param url
+ */
+export const canOpen = (url: string): boolean => {
+  return (
```

Comment:
How about relative URLs?
For example:
* //example.com
* /about-us
* ../index.html
* about-us/contact

Reply:
What do you suggest? :)

Reply:
Maybe we could use the [URL API](https://developer.mozilla.org/en-US/docs/Web/API/URL)

Reply:
Made some changes at [43869aa](https://github.com/genexuslabs/web-standard-functions/pull/57/commits/43869aa3f3e753fd264bb23930e122a67199505b), please take a look.